addons/l10n_sa_edi/wizard/account_move_reversal_view.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="view_account_move_reversal_inherit_l10n_sa_edi" model="ir.ui.view">
+            <field name="name">account.move.reversal.form.inherit.l10n_sa_edi</field>
+            <field name="inherit_id" ref="account.view_account_move_reversal"/>
+            <field name="model">account.move.reversal</field>
+            <field name="arch" type="xml">
+                <field name="reason" position="replace">
+                    <field name="country_code" invisible="1"/>
+                    <field name="reason" string="Reason" invisible="move_type == 'entry' and country_code != 'SA'"/>
```

Comment:
Shouldn't we also add a `required=...` for better usability ? :thinking:

Reply:
Thanks, indeed it would be weird to have the field not required in the view and then display an error when it's emtpy